egs/swbd/s5c/local/nnet3/tuning/run_tdnn_b.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# b is as a, but using a different splice indexes and geometrically increasing
+# relu dims across layers. It gives better results but takes 2.5x time for training
```

Comment:
does it really take 2.5x longer?  I thought you said in the separate thread that it took 50% longer.